Domain/ViewModel/TaskVm.cs

```diff
@@ -7,6 +7,7 @@ public class TaskVm
         public Guid Id { get; set; }
         public string Subject { get; set; }
         public Guid? AssignedMemberId { get; set; }
+        public MemberVm Member { get; set; }
```

Comment:
Since we have the whole `Member` we can remove `AssignedMemberId` I guess.
Also let's be consistent and rename it to `AssignedMember `